### DISCUSSION TITLE
Issue with performance

### DISCUSSION THREAD

**Discussion by buzzard10** [dani-garcia/vaultwarden#2273] [Q&A]
Hello Guys,

i have server with 4 GB of ram and 2 vCores where i'm running docker with voultwarden. I have few users and unfortunately i have few seconds of latency when i'm trying to load data in web interface. Is there any way to improve this?

I can't setup websocket, it can be connected?

**Comment by RealOrangeOne** [comment]
That will absolutely be more than enough to run vaultwarden. I suspect the slowdown is in the crypto in the client. I've seen large vaults take several seconds to open, but that's more an issue with the Bitwarden client (and possibly official server) than it is vaultwarden.

**Comment by buzzard10** [comment]
@RealOrangeOne thank you for your response. I have the same issue when i'm logging in to Bitwarden through web vault. Is there any chance to debug this? I have less than 10 users and maybe ~50 entries, ~20 organizations. It should work that way imho.

**Reply by RealOrangeOne** [reply]
Best thing to do is look at the network tab. If web responses are coming back fast, then it's not a vaultwarden problem.

See also this issue https://github.com/dani-garcia/vaultwarden/issues/1453

**Comment by buzzard10** [comment]
@RealOrangeOne 
here you have video. What do you think

https://user-images.githubusercontent.com/17282191/152683420-2884672e-a0d5-4ef3-8fa9-35ccaf89a7f9.mp4

?

**Comment by vitorbaptista** [comment]
I have a similar issue, made worse after I created an organization with 2000+ passwords. I takes 30s~1m for the web UI to become responsive after login. I wonder if there's a way to remove an Organization's passwords from the vault frontpage to avoid paying this cost upfront...

**Reply by BlackDex** [reply]
I think it is only possible if you make your self an Admin or Owner of that Organization and then select that you are not allowed to manage all collections, and disable the collection which contains all the passwords of that organization.

Since you are still and Admin or Owner, that would make you still able to show them if you go to the Organization it self.
